.circleci/config.yml

```diff
@@ -184,6 +206,15 @@ commands:
             cp ./failsafe_classname* ~/generated-tests/
             cp ./projects_with_path ~/generated-tests/
             cp /tmp/this_node* ~/generated-tests/
+      - run:
+          name: Compress Target Directories (Code Coverage)
+          when: always
+          command: |
+            .circleci/scripts/cc-save.sh
```

Comment:
Can we name the file a bit more meaningful, e.g. `codecoverage-save.sh`, or `coverage-save.sh`?

Reply:
Done